### DISCUSSION TITLE
Unneeded packages are being installed on fresh install

### DISCUSSION THREAD

**Discussion by mfazail** [nuxt/nuxt#19646] [Questions] [answered]
When i create a fresh nuxt3 project and install the dependencies(by default only nuxt) there are bunch of packages that get installed. I don't think i will be needing all hosting providers packages/adapters. 

![Screenshot_20230313_193322](https://user-images.githubusercontent.com/80394548/224727982-7b0e5fbe-7356-41e9-b052-acb90706b069.png)

**If there a way to exclude unneeded packages please let me know.**

**Comment by TheAlexLichter** [comment]
This might be a good issue for https://github.com/unjs/nitro

**Reply by mfazail** [reply]
Found a similar issue https://github.com/unjs/nitro/issues/82#issue-1202104701 
Hope this issue gets solved fast because the number of nuxt's dependencies is much and still its fast ☺️

**Reply by danielroe** [reply, marked as answer]
I believe this has been resolved upstream in https://github.com/unjs/unstorage/pull/183

**Reply by TheAlexLichter** [reply]
@danielroe For unstorage indeed, but not for nitro itself, right? 🤔
See e.g. https://github.com/unjs/nitro/blob/main/package.json#L48-L49
